HelloWorld/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,16 @@
+<manifest package="net.boondockradio.helloworld"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application android:allowBackup="true" android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name" android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+
+        <activity android:name=".MainActivity">
```

Comment:
When you add new Activity, you ALWAYS need to declare it to this `AndroidManifest.xml` otherwise the app crashes. (And I do this too often.)